src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/PullRequestPostAnalysisTask.java

```diff
@@ -73,20 +73,44 @@ public void finished(Context context) {
             LOGGER.warn("No pull request ID has been submitted with the Pull Request. Analysis will be skipped");
             return;
         }
-        ProjectAlmSettingDto projectAlmSettingDto = new ProjectAlmSettingDto();
 
         String enablePrDecoration = Optional.ofNullable(System.getenv("ENABLE_PR_DECORATION"))
                 .orElse(projectAnalysis.getScannerContext().getProperties().getOrDefault("sonar.analysis.enableprdecoration", "true"));
         LOGGER.info("sonar.analysis.enableprdecoration: {}", enablePrDecoration);
+
+        ProjectAlmSettingDto projectAlmSettingDto;
         Optional<AlmSettingDto> optionalAlmSettingDto;
         try (DbSession dbSession = dbClient.openSession(false)) {
-            projectAlmSettingDto.setAlmRepo(projectAnalysis.getScannerContext().getProperties().getOrDefault(
-					"sonar.pullrequest.github.repository", "Pay-Baymax/" + projectAnalysis.getProject().getKey()));
-            projectAlmSettingDto.setAlmSettingUuid("AXxy3BubdvWBwkcdvIfk");
-            projectAlmSettingDto.setAlmSlug("");
-            projectAlmSettingDto.setProjectUuid(projectAnalysis.getProject().getUuid());
-            projectAlmSettingDto.setSummaryCommentEnabled(Boolean.parseBoolean(enablePrDecoration));
-            optionalAlmSettingDto = dbClient.almSettingDao().selectByUuid(dbSession, "AXxy3BubdvWBwkcdvIfk");
+
+            Optional<ProjectAlmSettingDto> optionalProjectAlmSettingDto =
+                    dbClient.projectAlmSettingDao().selectByProject(dbSession, projectAnalysis.getProject().getUuid());
+
+            if (optionalProjectAlmSettingDto.isEmpty()) {
+                LOGGER.info("No ALM has been set on the current project, try using default hardcoded PayPay values");
```

Comment:
`try using` sounds like a action that should be taken by the reader of the log; is this the intention?
```suggestion
                LOGGER.info("No ALM has been set on the current project; using default hardcoded PayPay values");
```

Reply:
Moving forward I think we can require this to be set to remove any confusion what org will be used.

I think it's already set in all projects already? In that case we could already throw exception if it's not set.